packages/electrode-archetype-react-app/arch-clap.js

```diff
@@ -669,6 +685,12 @@ Individual .babelrc files were generated for you in src/client and src/server
       }
     },
 
+    "dev-es6": {
+      desc:
+        "Start your app with watch in development mode with webpack-dev-server with ENV_TARGET=es6",
+      task: ["dev-target-es6", "dev"]
```

Comment:
`task: "dev --target=es6"` should work also right?  No need for the extra `dev-target-es6` task.

Reply:
correction: `task: ["dev --target=es6"]`